pkg/rotator/rotator_test.go

```diff
@@ -334,6 +334,24 @@ func TestReconcileWebhook(t *testing.T) {
 				},
 			},
 		},
+		{
+			"apiservice-insecure-tls", APIService, nil, []string{"spec", "caBundle"}, &apiregistrationv1.APIService{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "v1alpha2.example.com",
+				},
+				Spec: apiregistrationv1.APIServiceSpec{
+					Group:                 "example.com",
+					GroupPriorityMinimum:  1,
+					Version:               "v1alpha2",
+					VersionPriority:       1,
+					InsecureSkipTLSVerify: true,
```

Comment:
nit but if this patch is accepted I'd argue for either putting a comment about this edge case here or modifying `testWebhook` for APIService to assert that this field is set to false after the CA was injected.

Reply:
good point!

Reply:
done! :)